src/mavlink-router/main.cpp

```diff
@@ -797,14 +797,9 @@ static int parse_confs(ConfFile &conf)
                 log_error("Expected 'port' key for section %.*s", (int)iter.name_len, iter.name);
                 ret = -EINVAL;
             } else {
-                if (validate_ip(opt_udp.addr) < 0) {
```

Comment:
What happens now if the ip/hostname is invalid? Does it crash somewhere, or does it just not work? Wouldn't it be nicer to have an error message similar to the one removed here?

Reply:
But then won't it fail in the `getaddrinfo` call (line 1169)? Shouldn't it print an error message (some `log_error()`) there? Right now it feels like it just does `exit(1)`, so we won't know what happened from the logs, right? Or did I miss something?